EFR32 build failed

**Describe the bug**
Not able to compile OpenThread for EFR32 target.

**To Reproduce**
Information to reproduce the behavior, including:
1. Build `master` branch using `efr32` makefile for board `BRD4161A`, like
    `make -f examples/Makefile-efr32 BOARD=BRD4161A`

**Expected behavior**
It should compile at least without any blocking errors.

**Console/log output**

```
 /ws/open-source/OpenThread/src/openthread-efr32/examples/../examples/platforms/efr32/radio.c:46:34: fatal error: em_openthread-system.h: No such file or directory
  #include "em_openthread-system.h"
                                   ^
compilation terminated.
make[5]: *** [libopenthread_efr32_a-radio.o] Error 1
make[5]: *** Waiting for unfinished jobs....
make[4]: *** [all-recursive] Error 1
make[3]: *** [all-recursive] Error 1
make[2]: *** [all-recursive] Error 1
make[1]: *** [all] Error 2
make: *** [do-build-cortex-m4] Error 2
```


**Additional context**
I think it's incorrect header file, it supposed to be **`#include "em_system.h"`**.

--- a/examples/platforms/efr32/radio.c
+++ b/examples/platforms/efr32/radio.c
@@ -43,7 +43,7 @@
 #include "utils/code_utils.h"
 
 #include "em_core.h"
-#include "em_openthread-system.h"
+#include "em_system.h"
 #include "openthread-core-efr32-config.h"
 #include "pa_conversions_efr32.h"
 #include "rail.h"

